apps/homepage/src/app/[lang]/_shared/menuButton/menuButton.tsx

```diff
@@ -0,0 +1,22 @@
+"use client";
+import styles from "./menuButton.module.scss";
+import Link from "next/link";
+import { motion } from "framer-motion";
+
+type menuButtonProps = {
+  text: string;
+  href: string;
+};
+
+export function MenuButton({ text, href }: menuButtonProps) {
```

Comment:
It would be great if this component is part of the `UI` package that we will add so we can export, publish and share these elements with other applications we create.